Amplify/Categories/Storage/Operation/Request/StorageGetURLRequest.swift

```diff
@@ -56,9 +56,11 @@ public extension StorageGetURLRequest {
         /// - Tag: StorageListRequestOptions.expires
         public let expires: Int
 
-        /// Extra plugin specific options, only used in special circumstances when the existing options do not provide
-        /// a way to utilize the underlying storage system's functionality. See plugin documentation for expected
-        /// key/values
+        /// Extra plugin specific options, only used in special circumstances when the existing options do
+        /// not provide a way to utilize the underlying storage system's functionality. See plugin
+        /// documentation or
```

Comment:
Hoping to update this with a link to the public docs once those changes are deployed.